pandas/plotting/_core.py

```diff
@@ -2835,18 +2835,37 @@ def area(self, x=None, y=None, **kwds):
 
     def pie(self, y=None, **kwds):
         """
-        Pie chart
+        Generate a pie plot.
+
+        A pie plot is a representation of the distribution of numerical data
+        in a DataFrame column. This function wraps the `matplotlib.pyplot.pie`
+        function for specified column.
 
         Parameters
         ----------
-        y : label or position, optional
-            Column to plot.
-        `**kwds` : optional
+        y : str or int, optional
+            Label or position.
```

Comment:
I find the `Column to plot` part important. What about `Label of position indicating the column to plot`?